bio/sexdeterrmine/wrapper.py

```
@@ -0,0 +1,44 @@
+    shell(f"sexdeterrmine --Input {depth_path} > out.tsv 2> sexdeterrmine.log")
+
+    log = snakemake.log_fmt_shell(
+        stdout=False,
+        stderr=True,
+        append=True,
+    )
+    shell("tree -shrf {tempdir} {log}")
+    os.chdir(old_path)
+    os.replace(f"{tempdir}/sexdeterrmine.log", str(snakemake.log))
+
+    tsv = snakemake.output.get("tsv")
```

Ruff (0.7.0)

Undefined name `snakemake`

(F821)